com.unity.multiplayer.mlapi/Runtime/Core/NetworkedBehaviour.cs

```diff
@@ -800,6 +800,7 @@ internal static void HandleNetworkedVarDeltas(List<INetworkedVar> networkedVarLi
                     long readStartPos = stream.Position;
 
                     networkedVarList[i].ReadDelta(stream, IsServer);
+                    PerformanceDataManager.Increment(ProfilerConstants.NumberNetworkVarsReceived.ToString());
```

Comment:
1) Would we not want to consolidate what's going on in PerformanceDataManager.Increment with ProfilerStatManager.networkVarsRcvd.Record.  They seem to be always together.

2) is there a way to avoid calling .ToString()?  The design at least with ProfilerStatManger was to yes record every count, but to do so super efficiently.  If we're doing a ToString() and then an allocation in the PerformanceDataManager.Increment call that seems like something I wouldn't without measuring be confident peppering around the SDK

Reply:
For #2, I'd suggest that rather than ProfilerConstants being an enum, make it a static class with string constants, something like this:

`
public static class ProfilerConstants {
    public const string NumberNetworkVarsReceived = nameof(NumberNetworkVarsReceived);
}
`
that way they are treated as literals and don't allocate at runtime

Reply:
No. ProfilerStatManager is going to be (eventually) moved out of MLAPI. Everything that its doing is what renderer's and logger's will be handling. 
I want parity of functionality and stat tracking before moving it up and out to an external area